### PR TITLE
nodejs-week1

### PR DESCRIPTION
**Pull request by patel-prakashkumar**

### CODE REVIEW

File: src/backend/app.js [hunk 1]
```diff
@@ -20,6 +21,53 @@ app.use(express.json());
 app.use(cors());
 
 router.use("/meals", mealsRouter);
+app.get("/my-route", (req, res) => { res.send("Hi friend") });
+
+app.get('/future-meals', async(req, res)=> {
+  const [row] = await knex.raw('SELECT * FROM Meal WHERE `when` > now()')
+  if (row.length === 0) {
+    res.status(404).json({ "Requested Meal ": "Not Availbel" })
```
Comment (itsgauravsaxena) [inline src/backend/app.js]: typo Availbel

**Review by itsgauravsaxena — APPROVED** [review]

File: src/backend/app.js (hunk 1 above)
Comment (patel-prakashkumar) [reply src/backend/app.js]: I have update typo mistake.

Comment (Shahnaz200) [reply src/backend/app.js]: nice job

**Comment by Shahnaz200** [comment]
good work

**Pull request merged**